scripts/reset_dev_data.py

```
@@ -41,278 +38,212 @@ class Colors:
+            WHERE table_schema = 'public' AND table_type = 'BASE TABLE'
+        """)
+        tables = [row[0] for row in cursor.fetchall()]
+
+        # Delete in order respecting FK constraints
+        delete_order = ["Session", "Account", "Project", "User"]
+        total_deleted = 0
+
+        for table in delete_order:
+            actual_table = table if table in tables else table.lower()
+            if actual_table in tables:
+                cursor.execute(f'DELETE FROM "{actual_table}"')
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)